### PR TITLE
Fix: Elements Regressions - Font "swap" make the tests fail [ED-8189]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: tests/elements-regression/env-setup/actions/change-font-display.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const ElementorSettingsPage = require( '../../pages/elementor-settings-page' );
+
+let elementorFontDisplayOriginalValue;
+
+module.exports = {
+	beforeAll: async ( { page } ) => {
+		const settingsPage = new ElementorSettingsPage( page );
```
Comment (matipojo) [inline tests/elements-regression/env-setup/actions/change-font-display.js:7]: Maybe just config it via `wp-cli` command?
https://github.com/elementor/elementor/blob/master/tests/wp-env/config/setup.sh#L10

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/elements-regression/env-setup/actions/change-font-display.js (hunk 1 above)
Comment (Nevoss) [reply tests/elements-regression/env-setup/actions/change-font-display.js:7]: But then it will affect only the CI, what about local testing?

Comment (matipojo) [reply tests/elements-regression/env-setup/actions/change-font-display.js:7]: It can be tested locally? (Linux screenshot)

Comment (Nevoss) [reply tests/elements-regression/env-setup/actions/change-font-display.js:7]: You can create your own photos + You can run on a container (e3 or `npm run test:elements-regression:update-snapshots`)

**Review by matipojo — CHANGES_REQUESTED** [review]
Please resolve the conflicts

**Review by matipojo — CHANGES_REQUESTED** [review]
Please resolve the conflicts

**Pull request closed without merging**